starboard/android/shared/media_capabilities_cache.cc

```diff
@@ -476,21 +476,17 @@ void MediaCapabilitiesCache::ClearCache() {
 
 void MediaCapabilitiesCache::ReloadSupportedHdrTypes() {
   ScopedLock scoped_lock(mutex_);
-  if (!is_initialized_) {
-    LazyInitialize_Locked();
-    return;
+  if (is_initialized_) {
```

Comment:
We need a more descriptive commit message to state the impact of this CL (i.e. what issue it resolves).

Reply:
(I am unable to put a comment in LazyInitialize_Locked(), so I leave it here)

Consider adding log to LazyInitialize_Locked() for how long it takes, it may help us debugging in future.

Also, assuming this causes ANR, which means that LazyInitialize_Locked() takes a few seconds, have we:
1. Take a run on the device to understand how long it takes?
2. Are we ok to push that latency to the first playback?  Will initializing this in a thread help?

Reply:
As discussed offline, changed the implementation to explicitly initialize the cache in background.